### PR TITLE
update icepyx to v0.8.0

### PR DESCRIPTION
**Pull request by JessicaS11**
@tsnow03 noted the hub image still used v0.7.0, which doesn't get s3 urls for all products.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ dependencies:
 
   # Packages specific to cryo work
   - h5coro~=0.0.6
-  - icepyx==0.7.0
+  - icepyx==0.8.0
```
Comment (weiji14) [inline environment.yml:122]: Minor suggestion to use `~=` (compatible release spec) per https://github.com/CryoInTheCloud/hub-image/issues/46#issuecomment-1542864196.

```suggestion
  - icepyx~=0.8.0
```

**Comment by weiji14** [comment]
Hmm, there seems to be an unrelated build failure at https://github.com/CryoInTheCloud/hub-image/actions/runs/6632090917/job/18017023795?pr=97#step:5:6493

```bash
   Could not solve for environment specs
  The following packages are incompatible
  ├─ jupyter-book 0.13.2  is installable and it requires
  │  └─ myst-nb >=0.13.1,<0.14.0 , which requires
  │     └─ myst-parser ~=0.15.2 , which requires
  │        └─ mdit-py-plugins ~=0.2.8 , which requires
  │           └─ markdown-it-py ~=1.0  with the potential options
  │              ├─ markdown-it-py 1.0.0 would require
  │              │  └─ attrs >=19.3,<21 , which can be installed;
  │              └─ markdown-it-py 1.1.0 would require
  │                 └─ attrs >=19.3,<22 , which can be installed;
  └─ referencing is not installable because it requires
     └─ attrs >=22.2.0 , which conflicts with any installable versions previously reported.
```

Seems to be an upstream issue, reported at https://github.com/executablebooks/jupyter-book/issues/2044.

File: environment.yml
Comment (weiji14) [inline environment.yml:1]: At L42, can you try upgrading `jupyter-book` to `~=0.15.1`? That should get rid of the attrs dependency conflict.

**Review by weiji14 — APPROVED** [review]
Thanks @JessicaS11!

**Comment by JessicaS11** [comment]
Thanks for the super fast review, @weiji14!

**Pull request merged**